src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/transport/action/addpolicy/TransportAddPolicyAction.kt

```diff
@@ -82,7 +127,12 @@ class TransportAddPolicyAction @Inject constructor(
                 .cluster()
                 .state(clusterStateRequest, object : ActionListener<ClusterStateResponse> {
                     override fun onResponse(response: ClusterStateResponse) {
-                        processResponse(response)
+                        val indexMetadatas = response.state.metadata.indices
```

Comment:
nit: Doesn't seem like this is used elsewhere, no need to instantiate a variable for it

Reply:
changed